cmd/ef_tests/levm/report.rs

```diff
@@ -751,7 +757,7 @@ impl fmt::Display for ComparisonReport {
                         writeln!(f, "      Storage slot added {levm_key} -> value mismatch REVM: {revm_value} LEVM: {levm_value}")?;
                         diffs += 1;
                     }
-                } else {
+                } else if *levm_key != H256::zero() && *levm_value != U256::zero() {
```

Comment:
Why are we checking for the key to be different than zero here?

Reply:
The report shows a difference when REVM doesn't store anything and we store the pair `(0,0)`.

However, these two have the same behavior at runtime, right?

That's why we are checking it; only show a difference if the values is different AND is not (0,0)

Reply:
Update: THis should only check the value, not the key. Plus, it is a bit of a border case, so it got removed here: https://github.com/lambdaclass/ethrex/pull/1519/commits/4883cc3eba64dd6fcde5030f4cac2c1d478944dd